### PR TITLE
Elaborate on service usage and functionality

### PR DESCRIPTION
**Pull request by george-angel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,53 @@
 # ingress53
 
-ingress53 is a service designed to run in kubernetes and maintain DNS records for its ingress resources on AWS Route53.
+ingress53 is a service designed to run in kubernetes and maintain DNS records for the cluster's ingress resources in AWS Route53.
+
+It will watch the kubernetes API (using the service token) for any Ingress resource changes and try to apply those records to route53 in Amazon mapping the record to the "target name", which is the dns name of the ingress endpoint for your cluster.
```
Comment (alkar) [inline README.md]: do we need to say "ingress controller" in the last sentence?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+
+# Requirements
+
+You need to export the following env variables to be able to use AWS APIs:
+
+```sh
+export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+exoprt AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```
Comment (alkar) [inline README.md]: typo, should be `export`

File: README.md [hunk 3]
```diff
@@ -17,6 +59,76 @@ You can test it locally (please refer to the command line help for more options)
...
+    spec:
+      containers:
+      - name: ingress53
+        image: utilitywarehouse/ingress53:v1.0.0
+        args:
+          - -route53-zone-id=XXXXXX
+          - -target-private=private.example.com
+          - -target-public=public.example.com
+          - -public-ingress-selector=ingress-tag-name:ingress-tag-value
+          - -debug
```
Comment (alkar) [inline README.md]: should we leave `debug` and `-v=10` out of it?

File: README.md (hunk 1 above)
Comment (george-angel) [inline README.md]: I don't think so, this does not need ingress controller to work, people can use something like `nodePort` and have this work for them.

File: README.md (hunk 3 above)
Comment (george-angel) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (george-angel) [inline README.md]: Done

**Pull request merged**